wiz-admission-controller/values.yaml

```diff
@@ -390,7 +390,9 @@ tlsCertificate:
   tlsCertificate: ""
   tlsKey: ""
 
-resources: {}
+# Remove the comment below to provide custom requests and limits to the Wiz pods.
+#
+# resources: {}
```

Comment:
```suggestion
# resources:
```

Reply:
Please make this easy to just uncomment without for it to work

Reply:
Done

Reply:
You didn't remove the {}

Reply:
You're right, apologies, now it's really done!